README.md

```diff
@@ -137,6 +137,9 @@ If you want to include snapshot releases in your project, you'll need to add the
    </dependency>
 ~~~
 
+## Projects based on this validator
+- The [French national access point to transport data](https://transport.data.gouv.fr/) is hosting a web interface of this validator available at [transport.data.gouv.fr/validation](https://transport.data.gouv.fr/validation?type=gtfs-rt). It can validate data using GTFS and GTFS Realtime URLs, and display the results.
+
```

Comment:
Here are a few more descriptions to add:
- [**CalTrans California Integrated Travel Project (Cal-ITP) GTFS Realtime validation**](https://dot.ca.gov/cal-itp/Critical-GTFS-Validation-Errors) - Cal-ITP is using the [gtfs-realtime-validator batch processor](gtfs-realtime-validator-lib/README.md#batch-processing), along with an open-source [Python wrapper they developed](https://github.com/cal-itp/gtfs-rt-validator-api), to validate GTFS Realtime data for transit agencies in the state of California.
- [**CUTR-at-USF's transit-feed-quality-calculator**](https://github.com/CUTR-at-USF/transit-feed-quality-calculator) - An open-source command-line Java application that uses the [gtfs-realtime-validator batch processor](gtfs-realtime-validator-lib/README.md#batch-processing) to assess the quality of a large number of transit feeds. It downloads a list of GTFS Realtime feed URLs from [OpenMobilityData](https://openmobilitydata.org/), fetches GTFS Realtime feeds from those agency URLs, runs the [gtfs-realtime-validator batch processor](gtfs-realtime-validator-lib/README.md#batch-processing) on all of the downloaded feeds, and then outputs summary statistics and graphs on the results.
- [**Traffic Brain (Japan)**](https://t-brain.jp/) - Runs [a hosted website version](https://gtfs.t-brain.jp/rt-validator/) of the [gtfs-realtime-validator-webapp](gtfs-realtime-validator-webapp/README.md) in this project.

Reply:
Perfect, thank you! I added them as well